cartography/data/indexes.cypher

```diff
@@ -64,3 +64,4 @@ CREATE INDEX ON :RDSInstance(id);
 CREATE INDEX ON :S3Acl(id);
 CREATE INDEX ON :S3Bucket(name);
 CREATE INDEX ON :User(arn);
+CREATE INDEX ON :ChromeExtensionPermission(name);
```

Comment:
Please index on `id` at minimum.  You can add an additional index for `name` if you expect other callers to query based on `name` and not `id`.

Also, please move this in alphabetical order.